bedrock/externalpages/templates/externalpages/pocket/includes/mobile-nav.html

```diff
@@ -0,0 +1,45 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+<div class="mobile-nav-wrapper" tabindex="-1">
+    <nav class="mobile-nav" role="dialog" aria-modal="true" aria-label="Pocket Mobile Menu">
+        <div class="mobile-nav-close">
+            <button class="mobile-nav-close-btn" data-tooltip="Close">Close the Pocket Mobile Menu</button>
+        </div>
+        <ul class="mobile-nav-list">
+            <li class="mobile-nav-list-item">
+                <a class="mobile-nav-list-link home" href="https://getpocket.com/home?src=navbar" id="global-nav-home-link">Home<span class="beta">BETA</span></a>
+            </li>
+            <li class="mobile-nav-list-item">
+                <a class="mobile-nav-list-link my-list" href="https://getpocket.com/my-list?src=navbar" id="global-nav-collections-link">My List</a>
+            </li>
+            <li class="mobile-nav-list-item">
+                <a class="mobile-nav-list-link discover" href="https://getpocket.com/explore?src=navbar" id="global-nav-discover-link">Discover</a>
+            </li>
+            <li class="mobile-nav-list-item">
+                <a class="mobile-nav-list-link collections" href="https://getpocket.com/collections?src=navbar" id="global-nav-collections-link">Collections</a>
+            </li>
+            <li class="mobile-nav-list-item">
+                <a class="mobile-nav-list-link archive" href="https://getpocket.com/my-list/archive">Archive</a>
```

Comment:
The items from Archive to All Tags don't have any IDs compared to the previous items. Do we want to add IDs to the rest of them, or remove all IDs since they're not in use (yet) anyway?

Reply:
i think i might just remove these id's since we're not using them for anything right now